### ISSUE TITLE
[UX] Hide redundant items in Node Edit form.

### ISSUE THREAD

**Issue by Graham-72** [backdrop/backdrop-issues#3705]
I have recently provided a user with access to edit some pages in a simple Backdrop site. I would have liked to hide the link 'Edit summary' because the pages to be edited do not have a summary, and I would have liked to remove the 'FORMATTING OPTIONS' control since in this case there are no options.

Can these items be hidden when they are not needed? 

![Screenshot_2019-04-23 Edit Page Useful links Kyson Fairways](https://user-images.githubusercontent.com/10559274/56561556-5d3d1380-659f-11e9-9c17-ebabf013782c.png)

**Comment by BWPanda** [comment]
I thought there was an option to hide the Summary field/link from the field/widget settings...

And as I posted in another, similar issue recently, while you're waiting for this feature to get into core, Simplify does just this (hides Formatting Options), and more ;-)

**Comment by olafgrabienski** [comment]
@Graham-72 - regarding the "Formatting options" link: In #3516 we changed the appearance of the link, and in #3595 we're trying to improve this section of the form, so that it makes also sense when there are no options. See a screenshot of the suggestions in https://github.com/backdrop/backdrop-issues/issues/3595#issuecomment-476147482. The issue has a working PR. Are you up to have a look at it and give your feedback?

**Comment by olafgrabienski** [comment]
To hide the "Edit summary" link, assuming your content type is "Page", go to *Page > Manage fields > Body > Configure*, and disable the checkbox "Summary input".

**Comment by klonos** [comment]
@Graham-72 since the body summary is part of a single field (of type `Text (long with summary)`), I don't see any easy way to hide it other than a `hook_form_alter`. I have done a quick search on how to achieve this is Drupal, and I was not able to find any contrib module that does that.

As for the formatting options, I believe that https://github.com/backdrop-contrib/simplify does this, but not sure if it does it per user role.

**Comment by klonos** [comment]
...teaches me to refresh my pages 😅

**Comment by Graham-72** [comment]
Thank you very much for your comments.

@olafgrabienski 

> To hide the "Edit summary" link, assuming your content type is "Page", go to Page > Manage fields > Body > Configure, and disable the checkbox "Summary input"

This is just what I needed to know! Hopefully I will remember in future.

I have looked at the PR for #3595 and have some concerns. I will take a more thorough look before commenting.

@BWPanda I tried using Simplify to hide the formatting options but it prevented the use of anything other than plain text. I will take another look soon and, if appropriate, raise an issue for Simplify.
